tests/fixtures/grpc.py

```diff
@@ -98,12 +120,9 @@ def health_check(host: str, port: int):
         health_check_request = health_pb2.HealthCheckRequest()
         stub.Check(health_check_request)
 
-    host = "localhost"
     wait_until(
-        lambda: health_check(host, grpc_server_port),
+        lambda: health_check(host, port),
         pause=0.5,
     )
 
-    yield grpc_server
-
-    grpc_server.stop()
```

Comment:
Why remove the stop? I would think we still need it when we test without docker support.

Reply:
NVM, there are two separate fixture.